Makefile

```diff
@@ -83,7 +83,10 @@ init: default
 ### run:              Start the apisix server
 .PHONY: run
 run: default
-ifeq ("$(wildcard logs/nginx.pid)", "")
+ifeq ("$(findstring /root, ${PWD})", "/root")
+	@echo "Non-zero exit status: 1"
```

Comment:
I mean to exit the Makefile with non-zero exit code.

Reply:
@spacewander 
Updated, please help me to see, is this right?